### PR TITLE
Release/v0.14.1

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/enums/CrossReferencePrefix.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package org.alliancegenome.curation_api.enums;
+
+public enum CrossReferencePrefix {
+	   MGI, ZFIN, SGD, FB, WB, XB, OMIM, GO
```
Comment (cmpich) [inline src/main/java/org/alliancegenome/curation_api/enums/CrossReferencePrefix.java]: Need to add 
RGD

File: src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java [hunk 2]
```diff
@@ -39,4 +43,17 @@ public class Reference extends InformationContentEntity {
...
+	 */
+	@Transient
+	@JsonIgnore
+	public String getReferenceID() {
+		Optional<CrossReference> opt = getCrossReferences().stream().filter(reference -> reference.getCurie().startsWith("PMID:")).findFirst();
+		// if no PUBMED ID try MOD ID
+		if (opt.isEmpty()) {
+			opt = getCrossReferences().stream().filter(reference -> CrossReferencePrefix.valueOf(reference.getPrefix()) != null).findFirst();
+		}
+		return opt.map(CrossReference::getPrefix).orElse(null);
```
Comment (cmpich) [inline src/main/java/org/alliancegenome/curation_api/model/entities/Reference.java]: This needs to be
		return opt.map(CrossReference::getCurie).orElse(null);

**Comment by oblodgett** [comment]
@cmpich do you want to make these changes?

**Review by cmpich — APPROVED** [review]
Looks good.

**Pull request merged**